src/libraries/System.IO.FileSystem/tests/FileStream/ReadWriteSpan.cs

```diff
@@ -24,6 +24,7 @@ public void DisposedStream_ReadWrite_Throws()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)] // PNSE: Cannot wait on monitors
```

Comment:
Don't we use `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]` for such a case?

Reply:
Addressed.  Thanks Maxim